NitroxServer-Subnautica/NitroxServer-Subnautica.csproj

```diff
@@ -1,115 +1,30 @@
-    <Compile Include="Serialization\Resources\Parsers\Text\EntityDistributionsParser.cs" />
-    <Compile Include="Serialization\Resources\Parsers\TextAssetParser.cs" />
-    <Compile Include="Serialization\Resources\Parsers\TransformAssetParser.cs" />
-    <Compile Include="Serialization\Resources\Processing\PrefabPlaceholderExtractor.cs" />
-    <Compile Include="Serialization\Resources\ResourceAssets.cs" />
-    <Compile Include="Serialization\Resources\ResourceAssetsParser.cs" />
-    <Compile Include="Serialization\SubnauticaServerJsonSerializer.cs" />
-    <Compile Include="Serialization\SubnauticaServerProtoBufSerializer.cs" />
-    <Compile Include="SubnauticaServerAutoFacRegistrar.cs" />
-  </ItemGroup>
-  <ItemGroup Label="Non-Build Files">
-    <None Include="App.config" />
-  </ItemGroup>
-  <ItemGroup Label="NuGet Packages">
-    <PackageReference Include="Autofac">
-      <Version>4.9.4</Version>
-    </PackageReference>
-    <PackageReference Include="Mono.Cecil">
-      <Version>0.11.3</Version>
-    </PackageReference>
-    <PackageReference Include="AssetsTools.NET">
-      <Version>2.0.6</Version>
-    </PackageReference>
-  </ItemGroup>
-  <ItemGroup Label="Project References">
-    <ProjectReference Include="..\NitroxModel\NitroxModel.csproj">
-      <Project>{b16f4de7-21ad-4fef-955b-0a5a365fa4e3}</Project>
-      <Name>NitroxModel</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\NitroxModel-Subnautica\NitroxModel-Subnautica.csproj">
-      <Project>{0a377218-6b36-4522-89a3-a39cfc999209}</Project>
-      <Name>NitroxModel-Subnautica</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\NitroxServer\NitroxServer.csproj">
-      <Project>{0fc864b0-694e-4fca-b78c-8ef98bc6f262}</Project>
-      <Name>NitroxServer</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup Label="Assembly References">
-    <Reference Include="protobuf-net">
-      <HintPath>..\Nitrox.Subnautica.Assets\protobuf-net.dll</HintPath>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="System.Drawing" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-</Project>
\ No newline at end of file
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net472</TargetFramework>
+        <OutputType>Exe</OutputType>
+        <RootNamespace>NitroxServer_Subnautica</RootNamespace>
+        <Nullable>disable</Nullable>
+        <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="..\NitroxModel-Subnautica\NitroxModel-Subnautica.csproj">
+            <Name>NitroxModel-Subnautica</Name>
```

Comment:
Is the extra defined name here required?

Reply:
It's not needed